### PR TITLE
Assign Press worker hosts

### PR DESCRIPTION
**Pull request by pumazi**
I unfortunately forgot to define `press_worker` in the inventory files and assign hosts to that host group. This rectifies that mistake.

### CODE REVIEW

**Comment by pumazi** [comment]
@Dantemss, that's the same thing I questioned in #cnx-deploy chat the other day. 😸 

And yep, no tests...

File: environments/content02/inventory [hunk 1]
```diff
@@ -54,32 +54,36 @@ content02
...
 publishing_worker
+press
+press_worker
 
 [db_connected:children]
 archive
 publishing
 channel_processing
 publishing_worker
-authoring
```
Comment (reedstrm) [inline environments/content02/inventory]: This is fine - authoring needs to be stripped out

File: environments/content02/inventory [hunk 2]
```diff
@@ -54,32 +54,36 @@ content02
...
 zclient
 pdf_gen
 backup
 press
+press_worker
 
 [nfs_connected:children]
-archive
```
Comment (reedstrm) [inline environments/content02/inventory]: This is not: we still need access to the NFS mounted file exports - archive uses FS access to list what downloadable files are available

File: environments/content03/group_vars/all/vars.yml [hunk 3]
```diff
@@ -16,8 +16,8 @@ press_db_user: "{{ archive_db_user }}"
 press_db_password: "{{ archive_db_password }}"
 press_db_host: "{{ archive_db_host }}"
 press_db_port: "{{ archive_db_port }}"
-press_broker_user: "{{ vault_press_broker_user }}"
```
Comment (reedstrm) [inline environments/content03/group_vars/all/vars.yml]: This too should probably stay vaulted

**Pull request merged**